core/templates/dev/head/pages/exploration_player/LearnerLocalNav.js

```diff
@@ -27,6 +27,8 @@ oppia.controller('LearnerLocalNav', [
       FocusManagerService, UrlInterpolationService,
       FLAG_EXPLORATION_URL_TEMPLATE) {
     $scope.explorationId = ExplorationEngineService.getExplorationId();
+    $scope.explorationVersion =
+    ExplorationEngineService.getExplorationVersion();
```

Comment:
Have you tested this?

I can see this method return GLOBALS.explorationVersion, so we cannot use it in place of `{% if exploration_version %}` as that jinja is defining GLOBALS.explorationVersion

Reply:
Can you please tell me why can we not use GLOBALS.explorationVersion? I can't understand how to solve it then.

Reply:
@DubeySandeep made some changes. PTAL!